docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -0,0 +1,370 @@
+
+      -  Generates the batch script and ancillary files to enable job submission (``sbatch``,
+         ``qsub``)
+
+      -  User-impersonation via a setuid program to enable the configured Determined user agent to
+         submit jobs to the cluster. Existing OS and workload manager permissions, and resource
+         enforcement is maintained. Launcher need not be configured to run as the root user.
+
+      -  Job submission and launch failure logs
+
+      -  On-demand job status query (``sinfo``, ``squery``, ``pbsnodes``, ``qstat``)
+
+      -  Job cancellation (``scancel``, ``qdel``)
+
+-  Support for various workload managers (Slurm/PBS) and container runtimes
+   (Apptainer/Singularity/Podman/Enroot)
+
+-  Provides an image caching model for Apptainer/Singularity
+
+-  Minimal state stored solely on the filesystem
+
+Job Management Overview
+=======================
+
+Temporary files (generated batch files, model python code, ``startup-hook.sh``, etc.), per-rank
+local directories in each container (``/tmp``, ``/run/determined/workdir``, etc.), and job log files
+are all stored under the cluster-wide directory specified via ``resource_manager.job_storage_root``.
+The lifetime of these files is limited to that of the submitted job, and are removed when job
+failure or completion is detected by Determined.
+
+#. Determined submits a job to the launcher.
+
+#. A directory tree is created by the launcher for the job.
+
+#. Files provided by Determined are staged.
+
+#. A batch script is generated to describe the required resource request (see Job Resource
+   Scheduling) from the workload manager and context (environment variables, and container launch
+   commands).
+
+#. The ownership of the files is transferred to the launching user (``sudo chmod``).
+
+#. A setuid image is executed to impersonate the launching user, which in turn executes the
+   sbatch/qsub command to submit the job.
+
+#. Upon being scheduled by the workload manager, the batch script is executed and tasks are launched on the allocated compute nodes.
+      #. One Determined environment image is started per node (``srun``, ``pbs_tmrsh``) using the
+         specified container runtime (Apptainer, Singularity, Podman, Enroot).
+
+      #. During container initialization, per-rank local directories are configured using soft links
+         (``/tmp``, ``/run/determined``), and the container stdout/stderr starts streaming output
+         directly to the Determined master.
+
+      #. For distributed training, horovodrun is invoked within the container to distribute work
+         within multiple processes across the containers.
+
+#. Once started, the job ID from the workload manager is displayed in the experiment log.
+
+#. Determined polls the launcher for job status waiting for completion.
+
+#. Determined requests the launcher to clean up the job. File ownership is transferred back to the
```

Comment:
Not sure if "Determined requests the launcher" is proper English.  Maybe "Determined sends a request to the launcher"?